railties/lib/rails/initializable.rb

```diff
@@ -60,13 +60,19 @@ def run_initializers(group = :default, *args)
       initializers.tsort_each do |initializer|
         initializer.run(*args) if initializer.belongs_to?(group)
       end
+      clear_initializers
```

Comment:
Would not this clean the initializers of a railtie before the other railties run?

Reply:
I'm not too sure yet, I'm still very much in the exploration process here. I doubt this approach will work in the end, but I'm trying to compute how much memory it would save first.